tests/foreman/ui/test_variables.py

```diff
@@ -913,62 +1511,66 @@ def test_positive_create_override_from_attribute(self):
         1.  The host/hostgroup is saved with changes.
         2.  New matcher for fqdn/hostgroup created inside variable.
 
-        @caseautomation: notautomated
+        @CaseLevel: Integration
         """
+        name = gen_string('alpha')
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            make_smart_variable(
+                session,
+                name=name,
+                puppet_class=self.puppet['name'],
+            )
+            self.assertTrue(self.smart_variable.validate_smart_variable(
+                name, 'overrides_number', '0'))
+            self.hosts.set_smart_variable_value(
+                self.host_name, name, gen_string('alpha'))
+            self.assertTrue(self.smart_variable.validate_smart_variable(
+                name, 'overrides_number', '1'))
 
     @run_only_on('sat')
-    @stubbed()
-    @tier1
+    @tier2
     def test_negative_create_override_from_attribute(self):
-        """No impact on variable on overriding the variable
-        with invalid value from attribute.
+        """No impact on variable on overriding the variable with invalid value
+        from attribute.
 
         @id: 18071443-a511-49c4-9ca9-04c7594b831d
 
         @steps:
 
         1.  Create a variable.
         2.  Associate variable with fqdn/hostgroup.
-        3.  From host/hostgroup, Attempt to override the variable with
-        some other key type of value.
+        3.  From host/hostgroup, attempt to override the variable with some
+        other key type of value.
 
         @assert:
 
         1.  Error thrown for invalid type value.
         2.  No matcher for fqdn/hostgroup is created inside variable.
 
-        @caseautomation: notautomated
+        @CaseLevel: Integration
         """
+        name = gen_string('alpha')
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            make_smart_variable(
+                session,
+                name=name,
+                key_type='integer',
+                default_value=gen_string('numeric'),
+                puppet_class=self.puppet['name'],
+            )
+            self.assertTrue(self.smart_variable.validate_smart_variable(
+                name, 'overrides_number', '0'))
+            self.hosts.set_smart_variable_value(
+                self.host_name, name, gen_string('alpha'))
+            self.assertIsNotNone(
+                self.hosts.wait_until_element(locators['host.override_error']))
+            self.assertTrue(self.smart_variable.validate_smart_variable(
+                name, 'overrides_number', '0'))
 
     @run_only_on('sat')
-    @stubbed()
-    @tier1
-    def test_negative_create_override_from_attribute_required_checked(self):
```

Comment:
`Required` checkbox functionality is not supported by Smart Variables and actually it is not even present on UI